components/motor/gpio/basic.go

```diff
@@ -214,7 +219,6 @@ func (m *Motor) setPWM(ctx context.Context, powerPct float64, extra map[string]i
 	}
 
 	powerPct = math.Max(math.Abs(powerPct), m.minPowerPct)
```

Comment:
[nit] I don't think we need this line anymore, since we moved the min logic to the top

Reply:
no but I'm not going to touch more in this pr. It will be merged in messy.

Reply:
Edit - no I'm msitaken. This is now the only thing making the pwmPinPower positive. otherwise the pin will crash when you set it to negative.